2024-02-19 10:10:51,759 DEBUG    root            Loaded Command Group: ['gcloud', 'config']
2024-02-19 10:10:51,761 DEBUG    root            Loaded Command Group: ['gcloud', 'config', 'config_helper']
2024-02-19 10:10:51,764 DEBUG    root            Running [gcloud.config.config-helper] with arguments: [--format: "json"]
2024-02-19 10:10:51,772 DEBUG    root            (gcloud.config.config-helper) Your current active account [default] does not have any valid credentials
Please run:

  $ gcloud auth login

to obtain new credentials.

For service account, please activate it first:

  $ gcloud auth activate-service-account ACCOUNT
Traceback (most recent call last):
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 815, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/surface/config/config_helper.py", line 99, in Run
    cred = store.Load(use_google_auth=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 723, in Load
    cred = _Load(
           ^^^^^^
  File "/usr/bin/../lib/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 911, in _Load
    raise NoCredentialsForAccountException(account)
googlecloudsdk.core.credentials.store.NoCredentialsForAccountException: Your current active account [default] does not have any valid credentials
Please run:

  $ gcloud auth login

to obtain new credentials.

For service account, please activate it first:

  $ gcloud auth activate-service-account ACCOUNT
2024-02-19 10:10:51,819 ERROR    root            (gcloud.config.config-helper) Your current active account [default] does not have any valid credentials
Please run:

  $ gcloud auth login

to obtain new credentials.

For service account, please activate it first:

  $ gcloud auth activate-service-account ACCOUNT
